.travis.yml

```diff
@@ -5,7 +5,8 @@ cache:
 notifications:
   email: false
 node_js:
-  - '9'
+  - "10"
+  - "9"
```

Comment:
Do we still need node 9? I think this will make our builds slower and I think we can drop it if we have everything on node 10.